graphql/database/missions/utils.js

```diff
@@ -0,0 +1,154 @@
+/* eslint-disable no-nested-ternary */
+/* eslint-disable no-param-reassign */
+import UserModel from '../users/UserModel'
+import {
+  getPermissionsOverride,
+  MISSIONS_OVERRIDE,
+} from '../../utils/permissions-overrides'
+
+const override = getPermissionsOverride(MISSIONS_OVERRIDE)
+export const buildSquadMemberDataFromMissionDoc = missionDoc => {
+  const {
+    acceptedSquadMembers,
+    pendingSquadMembersExisting,
+    rejectedSquadMembers,
+    pendingSquadMembersEmailInvite,
+  } = missionDoc
+  const squadMemberDataFromMissionDocAsDictionary = {}
+  acceptedSquadMembers.forEach(acceptedUserId => {
+    squadMemberDataFromMissionDocAsDictionary[acceptedUserId] = {
+      userId: acceptedUserId,
+      status: 'accepted',
```

Comment:
I recommend setting the statuses as constants at the top of this module as a good guard against typos.
```js
PENDING = 'pending'
ACCEPTED = 'accepted'
REJECTED = 'rejected'
```